src/lib/supported-browser.js

```diff
@@ -1,15 +1,34 @@
 import bowser from 'bowser';
 
+const minVersions = {
+    chrome: '63',
+    msedge: '15',
+    firefox: '57',
+    safari: '11'
+};
+
 /**
- * Helper function to determine if the browser is supported.
+ * Helper function to determine if the browser is supported at all.
  * @returns {boolean} False if the platform is definitely not supported.
  */
-export default function () {
-    if (bowser.msie ||
-        bowser.opera ||
-        bowser.silk) {
+const supportedBrowser = () => {
+    if (bowser.msie) {
```

Comment:
This could be `return !bowser.msie;`. I left it since there may be browsers other than IE that can't load gui at all.